test/functionalTest/cases/0117_convop_using_standard_ops/getEntityTypes.test

```diff
@@ -349,8 +349,8 @@ Date: REGEX(.*)
         {
             "attributes": [
 #SORT_START
-                "A2",
-                "A1"
+                "A2"REGEX(,?)
+                "A1"REGEX(,?)
```

Comment:
This "trick" is already used in this file (e.g. L367-368) but, for some reason, we forget to use it here...

NTC (just informative)